app/assets/config/manifest.js

```diff
@@ -0,0 +1,14 @@
+//= link_tree ../images
```

Comment:
I'm not quite sure this file is valid... Will need further testing.
For reference: https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs